src/ssh_fmt_plug.c

```diff
@@ -302,6 +302,7 @@ static void *get_salt(char *ciphertext)
 					ERR_print_errors_fp(stderr);
 					error();
 				}
+				return NULL;
```

Comment:
@frank-dittrich  It does not need to OPENSSL_free() when the function fails, since I searched some samples of using `PEM_read_bio`, such as: http://boinc.berkeley.edu/android-boinc/libssl/crypto/pem/pem_lib.c